### DISCUSSION TITLE
On the post list block when duplicating using MLP should we provide better user support

### DISCUSSION THREAD

**Discussion by carolinejcourtney** [amnestywebsite/humanity-theme#56] [Enhancement Requests]
When translators copy over a page from English, they need to reselect the posts in the post list.

**Comment by AmnestyAM** [comment]
Good idea, I think this might be big ish though. At some point it would be good to review that block as there are a lot of options and many are probably not used, would defiantly prefer to have few more useful options. Do you know how many times they have to do this and how long it takes each time?
